src/main/java/org/takes/http/MainRemote.java

```diff
@@ -53,11 +54,26 @@
     private final transient Class<?> app;
 
     /**
+     * Additional arguments to be passed to the main class.
+     */
+    private final transient String[] addargs;
```

Comment:
@HDouss just `args` is not enough? http://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html